doc/notebooks/XarrayforArviZ.ipynb

```diff
@@ -44,7 +44,10 @@
     "## Why not Pandas Dataframes or Numpy Arrays?\n",
     "Data from probabilistic programming is naturally high dimensional. To add to the complexity ArviZ must handle the data generated from multiple Bayesian Modeling libraries, such as pymc3 and pystan. This is an application that the *xarray* package handles quite well. The xarray package lets users manage high dimensional data with human readable dimensions and coordinates quite easily.\n",
```

Comment:
nitpick: 

* PyMC3 and PyStan

* and a few lines below where it says "the inspiration between *InferenceData*" I guess it should be "the inspiration for *InferenceData*"